aten/src/ATen/function_wrapper.py

```diff
@@ -548,15 +552,22 @@ def native_get_return_types(option):
         if ret['kind'] != 'type':
             raise Exception("native functions only support \'type\' return")
 
-        # can't actually return a TensorList (since it's a reference object)
-        actual_return_type = {'TensorList': 'std::vector<Tensor>'}.get(ret['type'], ret['type'])
-        if actual_return_type == 'Tensor' and option['inplace']:
-            # follow normal ATen convention of returning Tensor & for inplace functions.
-            actual_return_type = 'Tensor &'
-        return [{
-            'type': actual_return_type,
-            'dynamic_type': ret['type'],
-        }]
+        types = [x.strip() for x in ret['type'].split(',')]
```

Comment:
Maybe add a comment here indicating that you are handling multiple return types (it seems obvious now, but in general)